startup/10-machine.py

```diff
@@ -262,23 +28,109 @@ def stop(self, success=True):
 ANG_OVER_EV = 12.3984
 
 
+class InsertionDevice(Device, PositionerBase):
+    gap = Cpt(EpicsMotor, '-Ax:Gap}-Mtr',
+              kind='hinted', name='')
+    brake = Cpt(EpicsSignal, '}BrakesDisengaged-Sts',
+                write_pv='}BrakesDisengaged-SP',
+                kind='omitted', add_prefix=('read_pv', 'write_pv', 'suffix'))
+
+    # These are debugging values, not even connected to by default
+    elev = Cpt(EpicsSignalRO, '-Ax:Elev}-Mtr.RBV',
+               kind='omitted')
+    taper = Cpt(EpicsSignalRO, '-Ax:Taper}-Mtr.RBV',
+                kind='omitted')
+    tilt = Cpt(EpicsSignalRO, '-Ax:Tilt}-Mtr.RBV',
+               kind='omitted')
+    elev_u = Cpt(EpicsSignalRO, '-Ax:E}-Mtr.RBV',
+                 kind='omitted')
+
+    def set(self, *args, **kwargs):
+        set_and_wait(self.brake, 1)
+        return self.gap.set(*args, **kwargs)
+
+    def stop(self, *, success=False):
+        return self.gap.stop(success=success)
+
+    @property
+    def settle_time(self):
+        return self.gap.settle_time
+
+    @settle_time.setter
+    def settle_time(self, val):
+        self.gap.settle_time = val
+
+    @property
+    def timeout(self):
+        return self.gap.timeout
+
+    @timeout.setter
+    def timeout(self, val):
+        self.gap.timeout = val
+
+    @property
+    def egu(self):
+        return self.gap.egu
+
+    @property
+    def limits(self):
+        return self.gap.limits
+
+    @property
+    def low_limit(self):
+        return self.gap.low_limit
+
+    @property
+    def high_limit(self):
+        return self.gap.high_limit
+
+    def move(self, *args, moved_cb=None, **kwargs):
+
+        if moved_cb is not None:
+            @functools.wraps(moved_cb)
```

Comment:
`functools` may not be available here as it's imported inside of another method.